salesforcedx-49.13.1.sig

```diff
@@ -0,0 +1 @@
+bB9hDFYTiUwB9Io4ABr7ylfi3tRPf97msx0BG4xdgCyOs6tyaXrsv00RyMoGXfJEwwUcv4Dgb3djVZX5zt8NyuCBNQEW9MBv+OK5gTwZdyBAjCnlkrGEx7ptaQhXckEM2fqJrYSnQGFIcDwDNRGz59YCcqb81p5Ux7aXjjudit/qkjxsSRqhiskRBVZZWA+qGJbFQ9oTxYUTpKoTRSxtQkYG7p3h7PLVdVrmIyLzvW5NrB7ZC4+5PmYlId+J+rv0U5yXUg2+NfKnfHHMSnhtx3IttbWbA8uGgNzZN8CUcoI3GIe/1UJ1qNQEkKrwGxmhTT0tucBRkWvclfdzwcPk1g==
```

Comment:
Delete this file. It should never have been checked into the internal repo to begin with.